leases/tests/test_lease_mutations.py

```diff
@@ -1651,6 +1653,37 @@ def test_terminate_berth_lease_no_email_no_token(api_client):
     )
 
 
+@freeze_time("2020-12-01T08:00:00Z")
+@pytest.mark.parametrize(
+    "api_client", ["berth_services", "berth_handler"], indirect=True,
+)
+def test_terminate_berth_lease_before_season(api_client):
+    start_date = calculate_berth_lease_start_date()
+    end_date = calculate_berth_lease_end_date()
+
+    berth_lease = BerthLeaseFactory(
+        start_date=start_date,
+        end_date=end_date,
+        status=LeaseStatus.PAID,
+        application=BerthApplicationFactory(email="foo@email.com", language="fi"),
+    )
+
+    # Have to query to get the annotated value `is_available`.
+    assert not Berth.objects.get(id=berth_lease.berth_id).is_available
+
+    variables = {
+        "id": to_global_id(BerthLeaseNode, berth_lease.id),
+    }
+
+    executed = api_client.execute(TERMINATE_BERTH_LEASE_MUTATION, input=variables)
+
+    assert executed["data"]["terminateBerthLease"]["berthLease"] == {
+        "status": LeaseStatus.TERMINATED.name,
+        "endDate": str(start_date),
```

Comment:
Just to be 100% sure, could you also add 
```suggestion
        "startDate": str(start_date),
        "endDate": str(start_date),
```